api/openapi/openapi_metadata.py

```diff
@@ -0,0 +1,31 @@
+openapi_metadata = {
+    "title": "Observations Europe EUMETNET",
+    "description": (
+        "EDR data service for European environmental observations from EUMETNET," " co-funded by the European Union"
```

Comment:
Isn't "environmental observations" to broad?

Reply:
Good question. It is the only type of observations we have. I don't really have any better suggestions either.

Reply:
`... European meteorological observations ...`